aws-precise-staging-1/main.tf

```diff
@@ -80,14 +82,15 @@ module "aws_asg_com" {
   cyclist_scale = "web=1:Hobby"
   cyclist_version = "v0.1.0"
   env = "${var.env}"
+  env_short = "${var.env_short}"
   heroku_org = "${var.aws_heroku_org}"
   index = "${var.index}"
   security_groups = "${module.aws_az_1b.workers_com_security_group_id},${module.aws_az_1e.workers_com_security_group_id}"
   site = "com"
   syslog_address = "${var.syslog_address}"
   worker_ami = "${var.worker_ami}"
   worker_asg_max_size = "1"
-  worker_asg_min_size = "1"
+  worker_asg_min_size = "0"
```

Comment:
In reality the ASG will have at least one instance spin up automatically because of the scaling rules, but having a minimum of 0 allows us to set the desired capacity to 0 without modifying the ASG, which is something we need to do when tearing down the entire VPC.